### ISSUE TITLE
unable to go get reception

### ISSUE THREAD

**Issue by Vertikar** [ninech/reception#3]
```
karl@mini:~$ go version
go version go1.7.1 linux/amd64

karl@mini:~$ go env
GOARCH="amd64"
GOBIN=""
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/karl/go"
GORACE=""
GOROOT="/usr/local/go"
GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
CC="gcc"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build951816170=/tmp/go-build -gno-record-gcc-switches"
CXX="g++"
CGO_ENABLED="1"
```
```

karl@mini:~$ go get github.com/ninech/reception
# github.com/ninech/reception
go/src/github.com/ninech/reception/reception.go:121: frontend.Close undefined (type *"net/http".Server has no field or method Close)
```

Wondering if I need to be running a newer version of go? 
There wasn't mention in the readme of which version I needed.

**Comment by cimnine** [comment]
I think this has to do with the go version. I run this with go 1.9. I'll add it to the README.

**Comment by cimnine** [comment]
Fixed in da25ab0475e15e7532e2e28bdd4e61abe1b86bd1
